frontend/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -28,8 +35,18 @@ export const HorizontalTabs: Story = {
   render: (args) => {
     return (
       <Tabs {...args}>
-        <Tab onTabSelect={(id) => id} name="봄봄1" selected={false} />
-        <Tab onTabSelect={(id) => id} name="봄봄2" selected={true} />
+        <Tab
+          onTabSelect={(id) => id}
+          value="봄봄1"
+          selected={false}
+          label="봄봄1"
+        />
+        <Tab
+          onTabSelect={(id) => id}
+          value="봄봄2"
+          selected={true}
+          label="봄봄2"
+        />
```

Comment:
위에 meta에서 args를 넘겨주기 때문에 이 부분 다 삭제해도 돼요!

```suggestion
       <Tabs {...args} />
```